### PR TITLE
[Security solution] [Endpoint] Adds documentation for exceptions container summary

### PR DESCRIPTION
**Pull request by dasansol92**
Related issue: https://github.com/elastic/security-docs/issues/729

### CODE REVIEW

**Comment by jmikell821** [comment]
Hey there @dasansol92. Thanks again for helping out with this PR. Please confirm if I have the right table of contents order for this file. Preview link is [here](https://security-docs_824.docs-preview.app.elstc.co/guide/en/security/master/exceptions-api-summary-exception-container.html).

File: docs/detections/api/exceptions/api-summary-exception-container.asciidoc [hunk 1]
```diff
@@ -0,0 +1,64 @@
+[[exceptions-api-summary-exception-container]]
+=== Summary exception container
+
+Retrieves an exception container summary.
+
+NOTE: The {kib} Console supports only Elasticsearch APIs. You cannot interact with the {kib} APIs with the Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```
Comment (nastasha-solomon) [inline docs/detections/api/exceptions/api-summary-exception-container.asciidoc:6]:
```suggestion
NOTE: Console supports only Elasticsearch APIs. You cannot interact with {kib} APIs with Console and must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```

Comment (nastasha-solomon) [reply docs/detections/api/exceptions/api-summary-exception-container.asciidoc:6]: Checked the Kibana docs and it looks like they're using Console as a proper noun, which is why they don't use an article before it.

File: docs/detections/api/exceptions/api-summary-exception-container.asciidoc [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+==== Request body
+
+A JSON object with these fields:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`id` |String |Exception container id. |Yes
```
Comment (nastasha-solomon) [inline docs/detections/api/exceptions/api-summary-exception-container.asciidoc:20]:
```suggestion
|`id` |String |Exception container ID. |Yes
```

**Review by nastasha-solomon — APPROVED** [review]
Just two minor comments. Looks great overall, thank you!

**Review by joepeeples — APPROVED** [review]
Looks good; thank you!

**Pull request merged**